src/sssom/context.py

```diff
@@ -25,108 +22,79 @@
 )
 
 
-def get_jsonld_context():
-    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file.
-
-    [Auto generated from sssom.yaml by jsonldcontextgen.py]
-
-    :return: JSON-LD context
-    """
+def _get_jsonld_context():
+    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file."""
     with open(SSSOM_CONTEXT, "r") as c:
         context = json.load(c, strict=False)
-
     return context
 
 
-def get_extended_prefix_map():
-    """Get prefix map from bioregistry (obo.epm.json).
-
-    :return: Prefix map.
-    """
-    converter = Converter.from_extended_prefix_map(EXTENDED_PREFIX_MAP)
-    return {record.prefix: record.uri_prefix for record in converter.records}
+def get_internal_converter() -> Converter:
```

Comment:
No need for "internal" - "external" distinction. This control is established by controlling the OBO context.